src/languages/es.js

```diff
@@ -344,6 +344,7 @@ export default {
         linkHasBeenResent: 'El enlace se ha reenviado',
         weSentYouMagicSignInLink: ({loginType}) => `Hemos enviado un enlace mágico de inicio de sesión a tu ${loginType}.`,
         resendLink: 'Reenviar enlace',
+        validationCodeFailedMessage: 'Parece que ha habido un error con tu enlace de validación. O bien el enlace de validación ha caducado, o su cuenta no existe.',
```

Comment:
Just checking but did we run this translation past the Español team here at Expensify?

Reply:
The copy i received that we should use here is,
> Parece que hubo un error con el enlace de validación. El enlace de validación ha caducado o tu cuenta no existe.